jscomp/test/debug_mode_value.ml

```diff
@@ -1,15 +1,10 @@
-
-[@@@bs.config {
+[@@@mel.config {
   flags = [|
-  (* "-drawlambda"; *)
-  (* "-dlambda";  *)
-  (* "-dtypedtree"; *)
-  (* "-bs-diagnose" *)
```

Comment:
What were this for, debugging output while developing?

Reply:
I guess... a lot more in https://github.com/melange-re/melange/pull/793 too